### PR TITLE
Bump openssl to 1.1.1f

### PR DESCRIPTION
**Pull request by runlevel5**
## Changes

* Bump openssl to 1.1.1f

### CODE REVIEW

File: Formula/portable-openssl.rb [hunk 1]
```diff
@@ -26,13 +25,14 @@ def arch_args
       ["darwin64-x86_64-cc", "enable-ec_nistp_64_gcc_128"]
     else
       args = ["enable-md2"]
-      if Hardware::CPU.intel?
+      case
+      when Hardware::CPU.intel?
```
Comment (MikeMcQuaid) [inline Formula/portable-openssl.rb:29]: Leave this as-is.

**Comment by MikeMcQuaid** [comment]
This is failing CI everywhere.

**Comment by MikeMcQuaid** [comment]
@runlevel5 Can you try to test locally rather than just pushing commits to this PR? Thanks.

**Comment by runlevel5** [comment]
@MikeMcQuaid sure, it works locally for me. I will re-open the PR once I sort out the CI issue with my forked repo.

**Pull request closed without merging**